loose.json

```diff
@@ -42,22 +40,17 @@
         "space-infix-ops": 1,
         "space-unary-ops": 1,
         "wrap-iife": [2, "inside"],
-        "react/display-name": 1,
-        "react/jsx-boolean-value": 1,
-        "react/jsx-quotes": 1,
+        "react/jsx-boolean-value": [1, "never"],
+        "react/jsx-quotes": [1, "double", "avoid-escape"],
         "react/jsx-no-undef": 1,
-        "react/jsx-sort-props": 1,
-        "react/jsx-sort-prop-types": 1,
         "react/jsx-uses-react": 1,
         "react/jsx-uses-vars": 1,
         "react/no-did-mount-set-state": 1,
         "react/no-did-update-set-state": 1,
         "react/no-multi-comp": 1,
         "react/no-unknown-property": 1,
-        "react/prop-types": 1,
```

Comment:
this seems to duplicate what react does in dev mode.
TODO: is there a reason to use one vs. the other?  Should we leave this on for prod builds?